homeassistant/components/modbus/__init__.py

```diff
@@ -268,7 +267,6 @@
         vol.Optional(CONF_INPUT_TYPE, default=CALL_TYPE_REGISTER_HOLDING): vol.In(
             [CALL_TYPE_REGISTER_HOLDING, CALL_TYPE_REGISTER_INPUT]
         ),
-        vol.Optional(CONF_REVERSE_ORDER): cv.boolean,
```

Comment:
We normally deprecate a config option for two releases using `cv.deprecated`. Has that been done?

Reply:
Done.